src/ValueFormatters/QuantityFormatter.php

```diff
@@ -50,15 +61,17 @@ class QuantityFormatter extends ValueFormatterBase {
 	/**
 	 * @param DecimalFormatter|null $decimalFormatter
 	 * @param FormatterOptions|null $options
+	 * @param QuantityUnitFormatter $unitFormatter
 	 */
-	// FIXME: In all other ValueFormatters the FormatterOption parameter is the first one.
-	public function __construct( DecimalFormatter $decimalFormatter = null, FormatterOptions $options = null ) {
+	public function __construct( QuantityUnitFormatter $unitFormatter, DecimalFormatter $decimalFormatter = null, FormatterOptions $options = null ) {
 		parent::__construct( $options );
 
 		$this->defaultOption( self::OPT_SHOW_UNCERTAINTY_MARGIN, true );
 		$this->defaultOption( self::OPT_APPLY_ROUNDING, true );
+		$options->defaultOption( self::OPT_APPLY_UNIT, true );
 
 		$this->decimalFormatter = $decimalFormatter ?: new DecimalFormatter( $this->options );
+		$this->unitFormatter = $unitFormatter;
```

Comment:
You can avoid a breaking change with `$this->unitFormatter = $unitFormatter ?: new BasicQuantityUnitFormatter();`